.gitignore

```diff
@@ -6,3 +6,4 @@ phpunit
 composer.lock
 .DS_Store
 .idea/
+site
```

Comment:
What is `site`?

Reply:
@kenjis The site folder is generated with a build command. I believe the author of a PR builds the site locally, but with CI it is maybe not necessary to include it in gitignore.

Reply:
Should I remove it then?

Reply:
No problem. Keep it.